test/utils/p-retry.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const pRetry = require('p-retry')
+const timeout = require('./timeout')
+
+/*
+ * This is a wrapper for pRetry enabling an interval value
+ * i.e., the time to wait between retries, in milliseconds
+ */
```

Comment:
Maybe add a note that we can remove this if https://github.com/sindresorhus/p-retry/issues/27 gets resolved?